lib/commands/command-plugin-test.bash

```diff
@@ -129,10 +129,10 @@ plugin_test_command() {
     fi
 
     if [ -n "$plugin_command" ]; then
-      $plugin_command
+      "${plugin_command[@]}"
```

Comment:
Well, if you want to make asdf POSIX compliant in the long run, we might as well get rid of the array usage, since you'll have to do that at some point. I pushed a new commit doing so, and fixing some other quoting errors.